app/views/vm_common/_attach.html.haml

```diff
@@ -9,24 +14,22 @@
       .col-md-8
         = select_tag("volume_id",
                       options_for_select(@volume_choices),
-                      "ng-model"                    => "vmCloudModel.volume_id",
-                      :miqrequired                  => true,
-                      :checkchange                  => true,
-                      "selectpicker-for-select-tag" => "")
+                      "ng-model"                    => "vm.vmCloudModel.volume_id",
+                      :required                     => true,
+                      "selectpicker-for-select-tag" => true)
     .form-group
       %label.col-md-2.control-label
         = _('Device Mountpoint (optional)')
       .col-md-8
         %input.form-control{:type          => "text",
                             :name          => "device_path",
-                            'ng-model'     => "vmCloudModel.device_path",
-                            'ng-maxlength' => 128,
-                            :miqrequired   => false,
-                            :checkchange   => true}
+                            'ng-model'     => "vm.vmCloudModel.device_path",
+                            'ng-maxlength' => 128}
```

Comment:
This should be `maxlength`, `128` is a valid expression, but no need to use the ng- version with a constant...